src/transformers/modeling_bart.py

```diff
@@ -83,7 +83,7 @@
         tokenizer = BartTokenizer.from_pretrained('facebook/bart-large-cnn')
 
         ARTICLE_TO_SUMMARIZE = "My friends are cool but they eat too many carbs."
-        inputs = tokenizer([ARTICLE_TO_SUMMARIZE], max_length=1024, return_tensors='pt')
+        inputs = tokenizer.encode_plus(ARTICLE_TO_SUMMARIZE, max_length=1024, return_tensors='pt')
```

Comment:
I think we can still use `__call__`, but need to pass truncation=True.

```suggestion
        inputs = tokenizer([ARTICLE_TO_SUMMARIZE], truncation=True, return_tensors='pt')
```